tests/integration/components/leaflet-map-test.js

```diff
@@ -145,3 +146,17 @@ test('setting zoom to 0 should not throw', function(assert) {
 
   assert.equal(map._layer.getZoom(), 0, 'zoom 0 is set');
 });
+
+test('using bounds from lat-lng-bounds helper works', function(assert) {
+
+  this.set('markerCenter', locations.nyc);
+  this.set('bounds', locations.bounds());
+
+  this.render(hbs`
+    {{#leaflet-map bounds=(lat-lng-bounds bounds)}}
+      {{marker-layer location=markerCenter}}
+    {{/leaflet-map}}
+  `);
+
+  assert.ok(map._layer.getBounds() instanceof L.LatLngBounds);
```

Comment:
Here you're just testing that `getBounds` returns an instance of `L.LatLngBounds`. That test isn't meaningful.
We want to test that the bounds are what we expect them to be.

Please use the `boundsContain` helper, like here: https://github.com/miguelcobain/ember-leaflet/blob/master/tests%2Fintegration%2Fcomponents%2Fleaflet-map-test.js#L78